File: Payhub - SDK/ruby_SDK/PayHubSDK/lib/PayHubSDK/com/payhub/ws/vt/risk_fraud_settings.rb
class RiskFraudSettings
  include JsonSerializer
  ATTRS=[:transaction_volume_settings,:card_filtering,:email,:credit_card_security_codes,:address_verification_system,:errors]
  attr_accessor *ATTRS
  RISK_FRAUD_SETTINGS_LINK="adminSettings/riskFraud"
  RISK_FRAUD_PATCH_SETTINGS_LINK = "adminSettings/riskFraudDetection"
  def initialize
  end
end

class Transaction_volume_settings
  include JsonSerializer
  ATTRS=[:hours_trn_number_more_than,:days_trn_amount_more_than,:sale_trn_amount_below,:days_trn_number_more_than,:refund_trn_amount_above,:refund_trn_amount_below,:sale_trn_amount_above,:checked]
  attr_accessor *ATTRS
  def initialize
  end
end

class Card_filtering
  include JsonSerializer
  ATTRS=[:checked]
  attr_accessor *ATTRS

  def initialize
  end
end

class Email
  include JsonSerializer
  ATTRS=[:checked,:email_address]
  attr_accessor *ATTRS
  def initialize
  end
end
class Credit_card_security_codes
  include JsonSerializer
  ATTRS=[:checked,:cvv_mismatch]
  attr_accessor *ATTRS
  def initialize
  end
end
class Address_verification_system
  include JsonSerializer
  ATTRS=[:checked,:avs_mismatch_street_and_zip_code,:avs_mismatch_street,:avs_mismatch_zip_code]
  attr_accessor *ATTRS
  def initialize
  end
end

class OptionAndValue
  include JsonSerializer
  ATTRS=[:option,:value]
  attr_accessor *ATTRS
  def initialize
  end
end

class Hours_trn_number_more_than<OptionAndValue
end
class Days_trn_amount_more_than <OptionAndValue
end
class Sale_trn_amount_below <OptionAndValue
end
class Days_trn_number_more_than <OptionAndValue
end
class Refund_trn_amount_above <OptionAndValue
end
class Refund_trn_amount_below<OptionAndValue
end
class Sale_trn_amount_above<OptionAndValue
end
class Cvv_mismatch<OptionAndValue
end
class Avs_mismatch_street_and_zip_code<OptionAndValue
end
class Avs_mismatch_street<OptionAndValue
end
class Avs_mismatch_zip_code<OptionAndValue
end

File: Payhub - SDK/c#_SDK/PayHubSDK/PayHubSDK/com/payhub/ws/vt/RiskFraudSettings.cs
using PayHubSDK.com.payhub.ws.util;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Runtime.Serialization;
using PayHubSDK.com.payhub.ws.api;

namespace PayHubSDK.com.payhub.ws.vt
{
    [DataContract]
    public class RiskFraudSettings
    {
        public static string RISK_FRAUD_PATCH_SETTINGS_LINK = "adminSettings/riskFraudDetection";
        [DataMember]
	    public TrnVolSet transaction_volume_settings;
        [DataMember]
        public CardFiltering card_filtering;
        [DataMember]
        public RiskEmail email;
        [DataMember]
        public CreditCardSecurityCodes credit_card_security_codes;
        [DataMember]
        public Avs address_verification_system;
        [DataMember]
        public List<Errors> errors;
   
        [DataContract]
	    public class OptionAndValue{
            [DataMember]
            public float value;
            [DataMember]
            public int option;		
	    }
        [DataContract]
	    public class TrnVolSet{
            [DataMember]
            public OptionAndValue refund_trn_amount_below;
            [DataMember]
            public OptionAndValue hours_trn_number_more_than;
            [DataMember]
            public OptionAndValue days_trn_amount_more_than;
            [DataMember]
            public OptionAndValue sale_trn_amount_below;
            [DataMember]
            public OptionAndValue refund_trn_amount_above;
            [DataMember]
            public OptionAndValue days_trn_number_more_than;
            [DataMember]
            public OptionAndValue sale_trn_amount_above;
            [DataMember]
            public Boolean Checked;
		
	    }
        [DataContract]
	    public class CardFiltering{
            [DataMember]
            public Boolean Checked;
	    }
        [DataContract]
	    public class RiskEmail{
            [DataMember]
            public Boolean Checked;
            [DataMember]
            public string email_address;		
	    }
        [DataContract]
	    public class CreditCardSecurityCodes{
            [DataMember]
            public OptionAndValue cvv_mismatch;
            [DataMember]
            public Boolean Checked;		
	    }
        [DataContract]
	    public class Avs{
            [DataMember]
            public OptionAndValue avs_mismatch_street_and_zip_code;
            [DataMember]
            public OptionAndValue avs_mismatch_street;
            [DataMember]
            public OptionAndValue avs_mismatch_zip_code;
            [DataMember]
            public Boolean Checked;		
	    }
    }
}
